src/msg.rs

```diff
@@ -19,11 +19,11 @@ pub enum ExecuteMsg {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema, QueryResponses)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
-    #[returns(GetOwnersResponse)]
-    GetOwners {},
+    #[returns(GetConfigResponse)]
+    GetConfig {},
 }
 
 #[cw_serde]
-pub struct GetOwnersResponse {
+pub struct GetConfigResponse {
     pub owners: Vec<Owner>,
 }
```

Comment:
```suggestion
pub type GetConfigResponse = Vec<Owner>
```

I hate the struct way, its your preference.

Reply:
I'm not familiar with how to use the type vs the struct. I tried to swap it out, but it made breaking changes. Might try to refactor later, or revisit the idea in a new contract.